src/content/blog/v1.0_release.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "AutoEmulate v1.0.0 release"
+pubDate: "2025-07-31"
+description: "Updates on new features in v1.0.0 release"
+draft: true
+author: "AutoEmulate contributors"
+release: "v1.0.0"
+---
+
+🎉 We're excited to announce AutoEmulate's first major release with v1.0.0 🎉
+
+<br/><br/>
+This release sees the package fully integrated into the PyTorch ecosystem; PyTorch is one of the most popular open-source deep learning frameworks and one of the bedrocks of modern machine learning with Python.
+
+This release also contains new features, including methods for uncertainty quantification;  ensemble-based emulator models, which output predictions with uncertainty, and Bayesian model calibration, which enables inference of parameters likely to have generated observed data with uncertainty.
+
+<br/><br/>
+If you'd like to discuss any of our work on AutoEmulate or AI for Physical Systems, feel free to reach out to us at ai4physics@turing.ac.uk
+
+## What's new
+
+### PyTorch refactor and package redesign
+
+AutoEmulate was originally built on top of the scikit-learn framework, which is a great library for traditional machine learning tasks. However, as we moved towards more complex models and larger datasets, we found that we needed the flexibility and performance that PyTorch provides. The new PyTorch backend allows us to leverage GPU acceleration, making training and inference much faster. The PyTorch focus also allows AutoEmulate to integrate with other tools in the broader ecosystem, enabling end-to-end emulation workflows.
+
+### A new set of Emulator models
+
+AutoEmulate v1.0.0 includes a new set of Emulator models with a variety of different architectures. Some of these are wrapped implementations from PyTorch, Sklearn and elsewhere, but we also have a number of custom implementations including ensemble models where we quantify uncertainty by looking at the distribution over outputs. The table below summarises the models available in this release:
+
+| Emulator | Implementation | Backend | Multi-output support |
+|----------|----------------|---------|----------------------|
+| GaussianProcessExact | Custom | PyTorch | True |
```

Comment:
I wouldn't call the GPs custom, they are very much making use of GPytorch

Reply:
I can't decide what I think of making the custom/wrapper distinction this explicitly

Reply:
Perhaps better to remove that column, I guess more important thing I'm trying to show is that most v1 models are PyTorch

Reply:
Does this look right?

<img width="759" height="799" alt="Screenshot 2025-07-22 at 15 21 11" src="https://github.com/user-attachments/assets/f5eeab09-d610-4352-8edc-5a48349f7382" />